src/main/resources/view/ReminderWindow.fxml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.net.*?>
+<?import javafx.geometry.*?>
+<?import javafx.scene.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.image.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.stage.*?>
```

Comment:
Changing the imports to single explicit would be better, considering there is 2 css with similar style classes

Reply:
Understood I will rectify this .